test/verify/main.cpp

```diff
@@ -63,40 +63,58 @@ void validate_gpu(const migraphx::program& p, const migraphx::parameter_map& m)
     EXPECT(is_shared(ctx, p.get_context()));
 }
 
+// clang-format off
```

Comment:
We had to turn clang-format off here because of the differences in versions used between Windows and Linux, hip sdk, and dockers — it was impossible to satisfy all of them!

Reply:
formated manually to look clean